### PR TITLE
Update the deployment recommendation

### PR DESCRIPTION
**Pull request by queenypingcap**
Update the English document to be consistent with the Chinese version.

### CODE REVIEW

**Comment by queenypingcap** [comment]
@siddontang @shenli @iamxy PTAL.

File: op-guide/recommendation.md [hunk 1]
```diff
@@ -3,27 +3,53 @@ title: Deploying Recommendations
...
-| Component | # of CPU Cores | Memory | Disk Type | Disk | # of Instances |
-| ---- | ------- | -------- | ------- | ------- | --------- |
-| TiDB | 8+ | 16G+ |  |  | 2+ |
-| PD   | 8+ | 16G+ |  | 200G+ | 3+ |
-| TiKV | 8+ | 16G+ | SSD | 200G~500G | 3+ |
+| Component | Hardware Resources | # of CPU Cores | Memory | Disk Type | Disk  Size|
+| ---- |-------| ------- | -------- | ------- | ------- | 
+| TiDB | CPU, Memory | 8+ | 16G+ |  |  | 
+| PD   | CPU, Memory, Disk I/O |8+ | 16G+ |  | 200G+ |
+| TiKV | CPU, Memory, Disk I/O |8+ | 16G+ | SSD | 200G~500G |
```
Comment (siddontang) [inline op-guide/recommendation.md]: 32G+

**Comment by queenypingcap** [comment]
@siddontang @shenli @iamxy PTAL.

**Comment by queenypingcap** [comment]
@siddontang @shenli @iamxy PTAL.

File: op-guide/binary-deployment.md [hunk 2]
```diff
@@ -32,7 +34,9 @@ tar -xzf tidb-latest-linux-amd64.tar.gz
 cd tidb-latest-linux-amd64
 ```
 
-#### CentOS 6 (Not Recommended)
+### CentOS 6 (Not Recommended)
+
+**Warning:** It is not recommended to deploy on CentOS 6 because most of the development and tests are performed on the CentOS 7+ and Ubuntu 14.04+ platforms. There might not be enough tests on the CentOS 6 platform.
```
Comment (siddontang) [inline op-guide/binary-deployment.md]: developments

File: op-guide/binary-deployment.md [hunk 3]
```diff
@@ -139,20 +145,26 @@ Assume we have six machines with the following details:
     mysql -h 192.168.199.113 -P 4000 -u root -D test
     ```
 
-**Note:** If you are starting TiKV or deploying PD in the production environment, it is highly recommended to specify the path for the configuration file using the `--config` parameter. If the parameter is not set, TiKV or PD does not read the configuration file.
+**Note:** 
+- If you start TiKV or deploying PD in the production environment, it is highly recommended to specify the path for the configuration file using the `--config` flag. If the flag is not set, TiKV or PD does not read the configuration file.
```
Comment (siddontang) [inline op-guide/binary-deployment.md]: deploy PD

**Comment by siddontang** [comment]
Rest LGTM

**Comment by queenypingcap** [comment]
@siddontang @shenli PTAL.

File: op-guide/binary-deployment.md [hunk 4]
```diff
@@ -9,11 +9,13 @@ category: deployment
 
 A complete TiDB project contains PD, TiKV, TiDB. The start-up sequence is PD -> TiKV -> TiDB.
 
-To learn the TiDB architecture, see [TiDB Architecture](../README.md#TiDB-Architecture).
+Before your startt, see [TiDB Architecture](../README.md#TiDB-Architecture) and [Deployment Recommendations](op-guide/recommendation.md).
```
Comment (shenli) [inline op-guide/binary-deployment.md]: startt -> starting

File: op-guide/recommendation.md [hunk 5]
```diff
@@ -3,27 +3,53 @@ title: Deploying Recommendations
...
+| TiDB | 2+, highly available, and add more instances based on the concurrency and throughputs| 1+|
+| PD | 3+，highly available | 1+ |
+| TiKV | 3+，highly available，and add more instances based on the computing resources and storage capacity |3+|
 
-* Don’t deploy the PD instance and TiKV instance on the same disk
+**Note:** 
+
+- The TiDB instance can be deployed on its own, or share the same node with PD.
+
+- For PD and TiKV, deploy a single instance on one disk to avoid I/O conflict which impacts the performance.
```
Comment (shenli) [inline op-guide/recommendation.md]: conflict -> competition

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@siddontang PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**